recipe/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "iam-units" %}
```

Comment:
This file needs to be moved into `recipes/iam_units/meta.yaml`. Currently nothing is actually being built even though the tests all pass.

Reply:
Hm, okay. I can do this.

I was looking at examples like [this](https://github.com/conda-forge/ixmp-feedstock/tree/master/recipe) and [https://github.com/conda-forge/xarray-feedstock/tree/master/recipe](this) where the meta.yaml file is in `recipes/` directly.

Is there a reason there is a difference, e.g. the layout has changed recently? Where should I look to find out about these recent changes, whether I need to apply them to other recipes, etc.?

Reply:
Nothing has changed. The layout in `staged-recipes` is different than the feedstocks. Here you should be looking at:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml

Reply:
Thanks! It was the difference in layout that wasn't obvious.